docs/ide/how-to-increase-chances-of-performance-issue-being-fixed.md

```diff
@@ -238,6 +240,21 @@ tool can be found on the [Recording performance traces with
 PerfView](https://github.com/dotnet/roslyn/wiki/Recording-performance-traces-with-PerfView)
 page.
 
+## OOP Issues
```

Comment:
💡 If a particular version of Visual Studio adds these logs automatically, consider adding a note like the following:

```
> :bulb: Starting with Visual Studio 2019 version ______, out-of-process logs are automatically attached to feedback submitted using the Report a Problem tool.
```

If these logs are only attached following a recording, you could add the following sentence as well:

> To submit feedback using this functionality, use the Report a Problem tool and make sure to Record the problem steps before submitting the feedback.